dotCMS/src/main/java/com/dotmarketing/portlets/structure/action/EditFieldAction.java

```diff
@@ -318,6 +321,13 @@ private boolean _saveField(ActionForm form, ActionRequest req, ActionResponse re
                  }
             }
 
+            // Validate values entered for decimal/number/boolean type radio button field
+            if (field.getFieldType().equals(FieldType.RADIO.toString())){
+                if(!isValidRadioField(fieldForm,req)){
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)